pkg/host/hostname.go

```diff
@@ -0,0 +1,25 @@
+package host
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+// GetHostname returns the node hostname using kubernetes as lib to ensure that will obtain the same value
+func GetHostname() (string, error) {
```

Comment:
should we be using node/util.GetHostname()?

if we dont use it directly can you add a comment with a link to the code you borrowed this from?

Reply:
I did not want to add more deps to kurl only because of this piece of code. 
I will add the link for it.